src/mock/apis/staked-relayer.ts

```diff
@@ -19,6 +19,15 @@ export class MockStakedRelayerAPI implements StakedRelayerAPI {
         ];
     }
 
+    async map(): Promise<Map<AccountId, ActiveStakedRelayer>> {
+        const registry = new TypeRegistry();
+        const decodedAccountId = "0xd43593c715fdd31c61141abd04a99fd6822c8558854ccde39a5684e7a56da27d";
+        return new Map([
+            [new GenericAccountId(registry, decodedAccountId), <ActiveStakedRelayer>{ stake: new BN(10.2) as DOT }],
```

Comment:
`BN` does not support float, this will be truncated to 10.
Prefer to pass in strings in the constructor to avoid this type of issues, it would yield a proper error message.